transcend/resource_data_silo_test.go

```diff
@@ -94,19 +114,20 @@ func prepareDataSiloOptions(t *testing.T, vars map[string]interface{}) *terrafor
 	return terraformOptions
 }
 
-func deployDataSilo(t *testing.T, terraformOptions *terraform.Options) (types.DataSilo, []types.Plugin) {
+func deployDataSilo(t *testing.T, terraformOptions *terraform.Options) (types.DataSilo, []types.Plugin, types.DiscoClassScanConfig) {
```

Comment:
Changing the interface here required a lot of other things to change, but I do think it's the right move since discoClassScanConfigs are now just as much a part of dataSilos as plugins.